flask/app/analyses.py

```diff
@@ -23,60 +27,114 @@
 
 @analyses_blueprint.route("/api/analyses", methods=["GET"], endpoint="analyses_list")
 @login_required
-def list_analyses():
+@paged
+def list_analyses(page: int, limit: int) -> Response:
+    order_by = request.args.get("order_by", type=str)
+    allowed_columns = [
+        "updated",
+        "result_path",
+        "notes",
+        "analysis_state",
+        "assignee",
+        "requester",
+    ]
```

Comment:
`analysis_id` and `pipeline` should be allowed as well, where `pipeline` is a comma-separated list of `pipeline_id`'s (the frontend doesn't support this yet but I can update the filter component to just be a checkbox list of pipelines available).

Reply:
Can we just use the individual GET endpoint for `analysis_id`?

Reply:
I guess so. Meanwhile we can either hide the `analysis_id` column in the table, or disable filtering for that column.